specification/components/schemas/parameters/Organisation.yaml

```diff
@@ -0,0 +1,27 @@
+title: Details the organisation that was specified in a search criteria
+type: object
+required:
+  - name
+  - valueIdentifier
+properties:
+  name:
+    type: string
+    enum:
+      - organisation
+    example: 'organisation'
+    default: 'organisation'
+  valueIdentifier:
+    type: object
+    required:
+      - system
+      - value
+    properties:
+      system:
+        type: string
+        enum:
+          - 'https://directory.spineservices.nhs.uk/STU3/Organization'
+        example: 'https://directory.spineservices.nhs.uk/STU3/Organization'
+        default: 'https://directory.spineservices.nhs.uk/STU3/Organization'
+      value:
+        type: string
+        example: 'Cicero-Clinic'
```

Comment:
Would this not be an ODS code rather than the name of an organisation?

Reply:
updated to example ODS: G7H8J